### PR TITLE
Makes changes oranges requested in the PR but merged anyway because what the fuck is the value of my free time

### PR DESCRIPTION
**Pull request by harmonyn**
https://github.com/tgstation/tgstation/pull/25318

here's a novel concept, if you want a change made, then request it BEFORE merging

what the fuck is the 24-hour rule, what the fuck is my life that failing to do some unpaid volunteer coding while I was asleep constitutes getting bitched at

>merged 7 hours ago
>12:30 PM
>didn't code something at 5:30 AM wow guess I suck

### CODE REVIEW

**Comment by Ninja137** [comment]
Oh hey the fruit being a faggot again what a coincidence

**Comment by calzilla1** [comment]
>fruit
You mean the shit-tris?

**Comment by optimumtact** [comment]
Excuse me? I was pretty polite, there's no need to be so rude.

File: code/game/objects/items/devices/radio/headset.dm [hunk 1]
```diff
@@ -301,6 +301,8 @@
 	return
 
 /obj/item/device/radio/headset/AltClick(mob/user)
+	if(!istype(user) || !Adjacent(user) || user.incapacitated())
```
Comment (optimumtact) [inline code/game/objects/items/devices/radio/headset.dm]: you need to check isliving

Comment (optimumtact) [inline code/game/objects/items/devices/radio/headset.dm]: or change the typepath to mob/living

**Comment by bgobandit** [comment]
>"if we wait for swampbandit we will be here forever"

>"I was pretty polite, there's no need to be so rude"

spectacular memes

**Comment by optimumtact** [comment]
swampbandit why are you so angry

File: code/game/objects/items/devices/radio/headset.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/devices/radio/headset.dm]: is there a check for "does this user have hands"?

**Comment by ShadowDeath6** [comment]
more like dumbgobandit because he doesn't read minds and fix his shit before it's merged

File: code/game/objects/items/devices/radio/headset.dm (hunk 1 above)
Comment (bgobandit) [inline code/game/objects/items/devices/radio/headset.dm]: who says it requires hands, we already have hands-free/voice activated shit in 2017 let alone 2557

**Pull request merged**